tests/test_cases/test_cocotb/test_scheduler.py

```diff
@@ -189,9 +189,10 @@ async def test_kill_coroutine_waiting_on_the_same_trigger(dut):
     dut.clk.setimmediatevalue(0)
 
     victim_resumed = False
+    victim_wait = Event()
 
     async def victim():
-        await Timer(1)  # prevent scheduling of RisingEdge before killer
+        await victim_wait.wait()
```

Comment:
Did you mean to modify this test?

Reply:
Yes, the `Timer` wasn't the issue, that was just a way of forcing the killer coroutine to be scheduled before the victim coroutine. I changed it to an event because I felt it was more clear.

Reply:
What I mean is, didn't you intend to modify the test that came from _this_ PR, rather than a test that's already in master?

Reply:
(modifying both is fine, but it seems odd to just modify the unrelated one)

Reply:
Whoops... I guess I forgot which PR I'm on.